### PR TITLE
Block admin RSS feeds

### PR DESCRIPTION
**Pull request by andytson-inviqa**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -95,6 +95,7 @@
       '/media/downloadable' => 'restricted',
       '/pkginfo' => 'restricted',
       '/report/config.xml' => 'restricted',
+      '^/rss/(catalog/(notifystock|review)|order/new)(/|$)' => 'restricted',
```
Comment (hgajjar) [inline attributes/default.rb]: I think the starting "/" should be optional, as its also accessible without it.
Ex. "index.phprss/catalog/notifystock/"
Ref: https://support.hypernode.com/knowledgebase/how-to-protect-your-magento-store-against-brute-force/

Comment (andytson-inviqa) [inline attributes/default.rb]: that seems unlikely. Definitely not supported in nginx, and still would be weird in apache. I'll check tho. I've seemed to have missed off (/index\.php)? there tho

Comment (andytson-inviqa) [inline attributes/default.rb]: I've checked apache servers too and it doesn't work. I don't know why they think it'd work. Perhaps with bad server configuration only

Comment (hgajjar) [inline attributes/default.rb]: may be yes

**Comment by andytson-inviqa** [comment]
guess I'll change the CHangelog release date :)

**Pull request merged**